src/controller/artifact/abstractor.go

```diff
@@ -95,13 +96,13 @@ func (a *abstractor) abstractManifestV1Metadata(ctx context.Context, artifact *a
 		return err
 	}
 
-	digests := make([]string, len(manifest.FSLayers))
-	for i, fsLayer := range manifest.FSLayers {
-		digests[i] = fsLayer.BlobSum.String()
+	var ol q.OrList
+	for _, fsLayer := range manifest.FSLayers {
+		ol.Values = append(ol.Values, fsLayer.BlobSum.String())
```

Comment:
`q.OrList` here is very strange, suggest still use the digests here (change the type of element from string to interface{}), then make the value of the `digest` to `q.OrList` in the q.KeyWords.